src/components/Search/Search.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Search = ({ query, moviesFilter }) => (
+  <div className="field">
+    <label htmlFor="search-query" className="label">
+      Search movie
+    </label>
+    <div className="control">
+      <input
+        type="text"
+        id="search-query"
+        value={query}
+        className="input"
+        placeholder="Type search word"
+        onChange={event => (
+          moviesFilter(event.target.value)
+        )}
```

Comment:
не создавай новую функцию, передавай существующую
```suggestion
        onChange={moviesFilter}
```